lib/ecs_deploy/service.rb

```diff
@@ -180,6 +191,8 @@ def self.wait_all_running(services)
                 if s.deployments.size == 1 && s.running_count == s.desired_count
                   chunked_service_names.delete(s.service_name)
                 end
+                service = ss.detect {|sc| sc.cluster == File.basename(s.cluster_arn) && sc.service_name == s.service_name }
```

Comment:
`ss` is the services grouped by the same cluster, so I think the condition `sc.cluster == File.basename(s.cluster_arn)` is unnecessary:

```suggestion
                service = ss.detect {|sc| sc.service_name == s.service_name }
```